.gitignore

```diff
@@ -14,4 +14,7 @@ jspm_packages
 # notes
 *.txt
 
-generated/
\ No newline at end of file
+generated/
+
+.dccache
+.eslintrc.json
```

Comment:
This shouldn't be in git ignore since this file contains all the rules for the lintter

Reply:
Done

Reply:
Done